woodpecker/ember/app/.feature.yml

```diff
@@ -4,6 +4,7 @@ steps:
     settings:
       repo: ${CI_REPO}
       tags: "feature-${CI_COMMIT_BRANCH##feature/}"
+      purge: true
```

Comment:
Is this really needed? `true` is the default value according to the [docs](https://woodpecker-ci.org/plugins/Docker%20Buildx) and source:

https://codeberg.org/woodpecker-plugins/docker-buildx/src/commit/a1162f68460ccc917a0fec374459d1f1ad80c8ae/cmd/docker-buildx/config.go#L283

Reply:
Indeed, you are right. I'll drop the PR!